dhis-2/dhis-api/src/main/java/org/hisp/dhis/programrule/ProgramRuleActionType.java

```diff
@@ -120,4 +125,9 @@ public boolean isImplementable()
     {
         return NOTIFICATION_LINKED_TYPES;
     }
+
+    public static Set<ProgramRuleActionType> getServerSupportedTypes()
```

Comment:
Just make the Set public and remove get method.

Reply:
Getting your changes from the other PR